lib/src/working_copy.rs

```diff
@@ -867,8 +867,16 @@ impl TreeState {
 
             // TODO: Check that the file has not changed before overwriting/removing it.
             match diff {
+                Diff::Removed(TreeValue::GitSubmodule(_old_id)) => {
```

Comment:
This match can be moved into the next one, but it just felt cleaner to have it here